server.js

```diff
@@ -1,5 +1,3 @@
-process.env.UV_THREADPOOL_SIZE = 100;
```

Comment:
I see https://github.com/jtangelder/sass-loader/tree/fix/threadpool-issue. Maybe it will be fixed.